ScoutSuite/frontend/src/components/ResourceName/index.js

```diff
@@ -13,7 +13,7 @@ const propTypes = {
 };
 
 const defaultProps = {
-  renderData: data => data.name,
+  renderData: data => data ? data.name : '',
```

Comment:
We could return the property `name` with `lodash/get` , so it also returns an empty string, `data` exists but not `name`.

Reply:
Fixed